key-chord.el

```diff
@@ -183,6 +172,8 @@
 
 ;; ########   History   ########################################
 ;;
+;; 0.7 (2017-06-29) zk-phi
+;; 	Add option `key-chord-safety-interval'
```

Comment:
It makes little sense to release **in the middle** of a patch series. Also you should add the note about this new option in the commit that adds this new option. You can add a new history entry without releasing at the same time. Just use something like `Unreleased` instead of `0.7 (2017-06-29) zk-phi`.

Reply:
Thanks, I don't make a bump version in this PR.